values-it/strings.xml

```diff
@@ -1,142 +1,181 @@
+            <string name="label_select">Seleziona</string>
+            <string name="label_create">Crea</string>
+            <string name="text_freespace">%1$s liberi</string>
+            <string name="title_storage_chooser">Seleziona memoria</string>
+            <string name="title_storage_chooser_internal">Memoria interna</string>
+            <string name="label_select_external_storage">Seleziona memoria esterna</string>
+            <string name="create_folder_success">Nuova cartella creata</string>
+            <string name="create_folder_error">Impossibile creare la cartella</string>
+            <string name="create_folder_error_no_write_access">Impossibile scrivere in questa cartella</string>
+            <string name="create_folder_error_already_exists">La cartella esiste già</string>
+            <string name="hint_folder_name">Nome cartella</string>
+            <string name="error_tip_empty_field">Nome cartella vuota</string>
 
     <!--______Nav-Drawer______-->
-        <string name="label_links">Link</string>
+        <string name="label_links">Collegamenti</string>
         <string name="label_blog">Blog</string>
 
     <!--______GENERAL______-->
         <!--LABELS-->
             <string name="app_name">Open GApps</string>
+            <string name="label_ok">OK</string>
             <string name="label_cancel">Annulla</string>
             <string name="label_no">No</string>
             <string name="label_yes">Sì</string>
             <string name="label_close">Chiudi</string>
             <string name="label_adblock_detected">Rilevato Adblocker</string>
             <string name="label_permission_denied">Permesso negato</string>
-            <string name="permission_granted">Permesso consentito! Grazie!</string>
-            <string name="label_grant_permission">Concedi Permesso</string>
-            <string name="navigation_drawer_open">Apri navigation drawer</string>
-            <string name="navigation_drawer_close">Chiudi navigation drawer</string>
+            <string name="permission_granted">Permesso concesso! Grazie!</string>
+            <string name="label_grant_permission">Concedi permesso</string>
+            <string name="navigation_drawer_open">Apri drawer navigazione</string>
+            <string name="navigation_drawer_close">Chiudi drawer navigazione</string>
             <string name="label_accept">Accetta</string>
             <string name="label_accepted">Accettato</string>
             <string name="label_decline">Rifiuta</string>
             <string name="label_delete">Elimina</string>
-            <string name="title_optional_app_update">Aggiornamento App disponibile</string>
-            <string name="ms_prev">Precedente</string>
-            <string name="ms_next">Successivo</string>
-            <string name="ms_end">Completa</string>
+            <string name="label_delete_warning">Eliminare %1$s?</string>
+            <string name="dont_show_again">Non visualizzare più</string>
```

Comment:
It's actually the same, but "mostrare" is much more used than "visualizzare" in other translations of the same message, right? I mean on websites or other projects.